firmware/lib/config/config.h.template

```diff
@@ -3,20 +3,23 @@
 
 
 // ============= CONFIGURE THESE FIELDS BEFORE FLASHING ====================================================
-#define WIFI_SSID "WIFINAME" // Wifi name (please use 2.4gz network)
-#define WIFI_PASS "WIFIPASS" // Wifi password
 #define TIMEZONE_API_LOCATION "America/Vancouver" // Use timezone from this list: https://timezonedb.com/time-zones
-#define WEATHER_LOCAION "Victoria, BC" //city/state for the weather
-#define STOCK_TICKER_LIST "SPY,VT,GOOG,TSLA,GME" // Choose your 5 stocks to display on the stock tracker
-#define WEATHER_UNITS_METRIC //Comment this line out(or delete it) if you want imperial units for the weather
+#define WEATHER_LOCAION "Victoria, BC" // City, State for the weather
+#define STOCK_TICKER_LIST "SPY,VT,GOOG,TSLA,GME" // Choose your 5 stokcs to display on the stock tracker
+#define WEATHER_UNITS_METRIC // Comment this line out(or delete it) if you want imperial units for the weather
 #define FORMAT_24_HOUR false // Toggle 24 hour clock vs 12 hour clock, change between true/false
 #define SHOW_AM_PM_INDICATOR false // AM/PM on the clock if using 12 hour
 #define SHOW_SECOND_TICKS true // Ticking indicator on the centre clock
 #define INVERTED_ORBS false // Set to true if using InfoOrbs upside down. Inverts screens and re-orders screens and buttons.
-//#define WEB_DATA_WIDGET_URL "" // Use this to make your own widgets using an API/Webdata source
-//#define WEB_DATA_STOCK_WIDGET_URL "http://<insert host here>/stocks.php?stocks=SPY,VT,GOOG,TSLA,GME" // Use this as an alternative to the stock ticker widget
+//#define WEB_DATA_WIDGET_URL "" // use this to make your own widgets using an API/Webdata source
+//#define WEB_DATA_STOCK_WIDGET_URL "http://<insert host here>/stocks.php?stocks=SPY,VT,GOOG,TSLA,GME" // use this as an alternative to the stock ticker widget
```

Comment:
You probably changed newlines on these lines, since I see no other changes here?